sections/builder.js

```diff
@@ -1103,11 +1103,25 @@ export default [
     content: "ClearClauses",
     href: "Builder-clear"
   },
+  {
+    type: "method",
+    method: "clear",
+    example: ".clear(statement)",
+    description: "Clears the specified operator from the query. Avalilables: 'select' alias 'columns', 'with', 'select', 'columns', 'where', 'union', 'join', 'group', 'order', 'having', 'limit', 'offset', 'counter', 'counters'. Counter(s) alias for method .clearCounter()",
+    children: [
+      {
+        type: "runnable",
+        content: `
+        knex.select('email', 'name').from('users').where('id', '<', 10).clear('select').clear('where')
+        `
+      }
+    ]
+  },
   {
     type: "method",
     method: "clearSelect",
     example: ".clearSelect()",
-    description: "Clears all select clauses from the query, excluding subqueries.",
+    description: "Deprecated, use clear(statement). Clears all select clauses from the query, excluding subqueries.",
```

Comment:
Probably it would be clearer if you mentioned specific operation in descriptions, e. g. `use clear('select')`